projects/travel-tracker.md

```diff
@@ -149,76 +160,118 @@ password: travel
 
-* 4: Application has exceptionally well-factored code with little or no duplication.  The business-logic code driving functionality is cleanly separated from rendering, view-related code.   Excellent usage of `fetch` and updates DOM based on results of network requests.  Handles all scenarios for error handling.
-* 3: Application is thoughtfully put together with some duplication.  Application is organized into classes with some misplaced logic. Business-logic code is mostly separated from view-related code.  Great usage of `fetch` and updates DOM based on results in most scenarios, but may update DOM before a network request is complete.  Handles most scenarios for error handling.
-* 2: Class methods use a mix of array and object prototypes and for loops. Application runs but the code has long methods, unnecessary or poorly named variables, and needs significant refactoring.  Uses `fetch` effectively for `GET` but does not implement `POST`.  Has little error handling and only `console logs` errors if a network request fails.
-* 1:  Application generates syntax error or crashes during execution.  Application is not separated into classes and there is no separation of business-side logic and view-related code. Developer writes code with unnecessary variables, operations, or steps that do not increase clarity.
+While M2 rubrics do not have a separate section for WOWs like in M1, there are a few WOW examples noted throughout. In addition to these WOW bullets, you can strive for a WOW by demonstrating not just competency, but excellence and thoroughness across the rubric sections.
+</section>
 
+<section class="answer">
+### Functional Expectations
 
-## Testing
+- Wow: Application fulfills all requirements as well as Iteration 5 or an extension.
+- Yes: Application fulfills all requirements of iterations 1-4 without bugs. **Note: Must be completed in order to pass**
+- Not Yet: Application crashes or has missing functionality or bugs.
+</section>
+
+<section class="answer">
+### UI/UX & Accessibility
+
+* The application can stand on its own to be used by an instructor without guidance from the developer
+* UI/UX is intuitive and easy to read/use
+* Helpful messaging is displayed to prevent user confusion. For example: If a user searches for a room, but none are available then a message is displayed to indicate that the search worked, nothing is broken, there just aren't any matching rooms available.
+* The Lighthouse audit tool was used to improve accessibility.
+* Users who only use keyboards can still navigate this application
+* Users who use a screen reader can still navigate this application
+* Utilizes ARIA attributes on interactive elements when no other tool allows for accessible content
+* Wow option: design is responsive across small, medium and large breakpoints
```

Comment:
I'm curious if it would help to section off what is noted for "On Track" versus "Wow" similar to how some of the Mod 1 projects are done.  [Example here](https://frontend.turing.edu/projects/module-1/colorandom.html). This is mainly from a readability standpoint.

Reply:
We had a big, long discussion last week on this and what was decided on was this format. Would you feel comfortable if we moved forward with this style and then if it feels weird, the team can update it? I see the value of both routes - but I'm not sure opening discussion up again on this topic will net a different result this time.